src/main/java/com/cb/dao/DaoImpl/UserDaoImpl.java

```diff
@@ -20,48 +21,62 @@ public void setTemplate(JdbcTemplate template) {
 
     @Override
     public List<UserBean> getUsers() {
-        return  template.query("SELECT * FROM users", new RowMapper<UserBean>() {
+        return template.query("SELECT * FROM users", new RowMapper<UserBean>() {
             @Override
             public UserBean mapRow(ResultSet rs, int rowNum) throws SQLException {
-            UserBean u = new UserBean();
-            u.setId(rs.getInt(1));
-            u.setUserName(rs.getString(2));
-            u.setPassword(rs.getString(3));
-            u.setAdmin(rs.getByte(4));
-            u.setRating(rs.getInt(5));
-            u.setMoney(rs.getInt(6));
-            u.setCharacterId(rs.getInt(7));
-            u.setPartyId(rs.getInt(8));
-            return u;
+                UserBean u = new UserBean();
+                u.setId(rs.getInt(1));
+                u.setUserName(rs.getString(2));
+                u.setPassword(rs.getString(3));
+                u.setAdmin(rs.getByte(4));
+                u.setRating(rs.getInt(5));
+                u.setMoney(rs.getInt(6));
+                u.setCharacterId(rs.getInt(7));
+                u.setPartyId(rs.getInt(8));
+                return u;
             }
         });
     }
 
     @Override
     public int insertUser(UserBean u) {
         String sql = "INSERT INTO users(userName, password, email, isAdmin, rating, money, characterId, partyId) " +
-            "VALUES ('"+u.getUserName()+"','"+u.getPassword()+"','"+u.getEmail()+"',"+u.getAdmin()+","+u.getRating()+","+u.getMoney()+","
-            +u.getCharacterId()+","+u.getPartyId()+")";
+                "VALUES ('" + u.getUserName() + "','" + u.getPassword() + "','" + u.getEmail() + "'," + u.getAdmin() + "," + u.getRating() + "," + u.getMoney() + ","
+                + u.getCharacterId() + "," + u.getPartyId() + ")";
         return template.update(sql);
     }
 
+    @Override
+    public List<UserBean> getUserByEmail(String email) {
```

Comment:
Here is the main part of checking user by email, yes? Cause I'll need to make the login feature

Reply:
Yes this checks by email